src/react-native-selectize.js

```diff
@@ -5,7 +5,7 @@ import { StyleSheet, Text, TextInput, TouchableOpacity, View, ViewPropTypes } fr
 import PropTypes from 'prop-types';
 import { normalize, schema } from 'normalizr';
 import Chip from './react-native-chip';
-
+import Label from './label'
```

Comment:
What this line refer to? I cannot find that file.

Reply:
@althurzard did you forget to add the Label component `index.js` to your commit?

Reply:
oops, i forgot adding label folder :D. I'll create another pull request later.